yt/chyt/controller/internal/agent/agent.go

```diff
@@ -407,12 +408,12 @@ func (a *Agent) background() {
 	}
 }
 
-func (a *Agent) getOpletInfoForScaler() []strawberry.OpletInfoForScaler {
-	opletInfos := make([]strawberry.OpletInfoForScaler, 0, len(a.aliasToOp))
+func (a *Agent) getOpletInfoForScaler() []strawberry.OpletInfosForScaler {
```

Comment:
A little bit of misunderstanding here :)
My comment was about `getOpletInfoForScaler` naming.
One oplet info is `OpletInfoForScaler`, withous **s**, but since this function returns an array of `OpletInfo` structs, it should be called `getOpletInfos...` with **s**

Reply:
Oh, shame on me. I'll fix it asap